src/agent/agent.c

```diff
@@ -26,6 +26,9 @@
 #define DEBUG_SYSTEMD_MESSAGES 0
 #define DEBUG_SYSTEMD_MESSAGES_CONTENT 0
 
+char *sections_that_can_be_reloaded[] = {};
```

Comment:
Wouldn't it be better to set to `NULL` by default?

Reply:
Done